### ISSUE TITLE
Definition::getSqlDefinition missing default values

### ISSUE THREAD

**Issue by ErikZigo** [keboola/php-datatypes#17]
Nevim jestli je to feature, nebo bug.
Vsechny typy s vyjimkou `GenericStorage` ignoruji nastaveni default hodnoty a nevraci ho v metode `getSqlDefinition()`

**Comment by pivnicek** [comment]
jo, ten `getSqlDefinition()` tam ma vratit zatim `NotImplementedException` nebo neco takhleho.  Ani nevim jestli generic Sql definition ma smysl pro GenericStorage types.

**Comment by ErikZigo** [comment]
```
    /**
     * @return string
     */
    public function getSQLDefinition()
    {
        $definition =  $this->getType();
        if ($this->getLength() !== null && $this->getLength() !== "") {
            $definition .= "(" . $this->getLength() . ")";
        }
        if (!$this->isNullable()) {
            $definition .= " NOT NULL";
        }
        return $definition;
    }
```

@pivnicek vubec ti nerozumim :-)

@ondrejhlavacek vsiml jsem si ze ty to vyuzivas v Transformacich.

Tam ale v UI se default hodnoty nastavovat nedaji. Takze kdyby se to dodelalo, tak by ti to nic nemelo rozbit ne?

**Comment by pivnicek** [comment]
@ErikZigo :-)

jo, 

> Takze kdyby se to dodelalo, tak by ti to nic nemelo rozbit ne?

ano

**Comment by ondrejhlavacek** [comment]
Pokud ty defaulty nebudou mít defaulty, tak to bude v pohodě :-)
